src/services/activity/activity.service.ts

```diff
@@ -1,17 +1,24 @@
-import _ from "lodash"
-import { ActivityType, ClientUser } from 'discord.js'
+import { ActivityType, ClientUser } from "discord.js";
+import _ from "lodash";
 
 export class ActivityService {
-  private static _instance: ActivityService
+  private static _instance: ActivityService;
 
   public static getInstance(): ActivityService {
-    if(_.isNil(ActivityService._instance)) {
-      ActivityService._instance = new ActivityService()
+    if (_.isNil(ActivityService._instance)) {
+      ActivityService._instance = new ActivityService();
     }
-    return ActivityService._instance
+    return ActivityService._instance;
   }
 
-  public setActivity(user: ClientUser, activity: ActivityType) {
-    user?.setActivity("!hell help", { type: activity })
+  public setActivity(user: ClientUser, activity: ActivityType): void {
+    user
+      .setActivity(`!hell help`, { type: activity })
+      .then((): void => {
+        console.info(`The bot activity was set to: ${activity}`);
```

Comment:
As said on Discord I really think that this is a bad idea to have such logs outside of the methods.
Reading the messages this is clear something that is describing the state of the setActivity method and then it should be within it.

Also why is this something bad from my opinion?
Let's say that later this method is called 30 times in your code.
For each of these calls it makes sense to log this.
Then you will have 30 duplications.

Reply:
I see. SO I need to remove all the logs I've put for Discord functions? ^^

Reply:
@asmodee-bellanie Not removing, only changing where you put them.

Reply:
Ok. And that means?

Reply:
Put the log in the called method.
In this case setActivity